### PR TITLE
Bump nixpkgs (rebased)

### PR DESCRIPTION
**Pull request by dfordivam**
This contains parts of #33 and #34

### CODE REVIEW

File: build-wasm-app.nix [hunk 1]
```diff
@@ -26,6 +26,7 @@ let
 in { ename, pkg, assets ? [], scripts ? [], styles ? [] }: runCommand "wasm-app-${ename}" {
   nativeBuildInputs = [ buildPackages.xorg.lndir buildPackages.binaryen ];
   passthru = { inherit pkg; };
+  meta.platforms = ["wasm32-unknown"];
```
Comment (ElvishJerricco) [inline build-wasm-app.nix:29]: ~We should see if we can remove this.~

Comment (ElvishJerricco) [reply build-wasm-app.nix:29]: Scratch that. Makes sense now I think

File: cross-overlays.nix [hunk 2]
```diff
@@ -9,7 +9,7 @@ self: super: {
 
   haskell = let inherit (super) haskell; in haskell // {
     packages = haskell.packages // {
-      ghcWasm = haskell.packages.ghc863.override (drv: {
+      ghcWasm = haskell.packages.ghc865.override (drv: {
```
Comment (ElvishJerricco) [inline cross-overlays.nix:12]: Should this be 865, or 881?

Comment (dfordivam) [reply cross-overlays.nix:12]: This was later changed to 881 in the ghc bump PR

**Pull request merged**